source/gloperate-glfw/source/Window.cpp

```diff
@@ -472,25 +471,25 @@ void Window::handleEvent(WindowEvent & event)
 
 void Window::clearEventQueue()
 {
-    while (!m_eventQueue.empty())
-    {
-        delete m_eventQueue.front();
-        m_eventQueue.pop();
-    }
+    std::queue<std::unique_ptr<WindowEvent>> empty;
```

Comment:
I think I misinterpreted the red background and minus signs on the previous lines; they actually aren't part of the source code after the merge. Thus, your code is a correct substitution of the former code.